test/system/src/main/java/io/pravega/test/system/framework/kubernetes/K8sClient.java

```diff
@@ -185,16 +185,31 @@ public V1Namespace createNamespace(final String namespace) {
                        });
     }
 
-
     /**
      * Method to fetch the status of all pods which match a label.
      * @param namespace Namespace on which the pod(s) reside.
      * @param labelName Name of the label.
      * @param labelValue Value of the label.
      * @return Future representing the list of pod status.
      */
-    @SneakyThrows(ApiException.class)
     public CompletableFuture<List<V1PodStatus>> getStatusOfPodWithLabel(final String namespace, final String labelName, final String labelValue) {
+        return getPodsWithLabel(namespace, labelName, labelValue)
```

Comment:
Even if we are using only one label at the moment, this method should support multiple labels to cover future usages.

Reply:
Created a new method.